submit.go

```diff
@@ -487,6 +484,23 @@ func PrintLog(jobLog []*Log) (lastPrintLog uint64) {
 	return lastPrintLog
 }
 
+// Process the exit code depending on the job status
+func getExitCode(job *JobStatus, notCompletedExitCode int64) int {
+	log.Printf("Job status is : %s", job.Status)
+	codeToReturn := job.ReturnCode
+
+	switch job.Status {
+		case JobStatusCOMPLETED:
+			log.Printf("Job exit code : %v", job.ReturnCode)
+		case JobStatusTERMINATED, JobStatusFAILED:
+			log.Printf("Job is finished, but not completely, fixed exit code : %v", notCompletedExitCode)
```

Comment:
```suggestion
			log.Printf("Job have been interrupted, spark-submit exiting with code : %s", notCompletedExitCode)
```

Reply:
Ty, it's taken into account. Why change `%v` to `%s` (in the original code `%v` was used), maybe even use `%w`? (https://stackoverflow.com/a/61287626)

Reply:
@Gudsfile you are right `%s` and `%v` are identical. Thanks for the link